sources/packages/backend/apps/db-migrations/src/sql/SFASRestrictions/Add-col-is-included.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+  sims.sfas_restrictions
+ADD
+  COLUMN processed BOOLEAN NOT NULL DEFAULT FALSE;
+
+COMMENT ON COLUMN sfas_restrictions.processed IS 'This field indicates whether the SFAS restriction has been added to the student restrictions table or not.';
```

Comment:
The comment must be adjusted. Not every single record will be added to the student restrictions.
Can you adjust to something like below?
```
Indicates if an SFAS restriction record was verified to determine if a restriction must be created in the student account.
```